src/test/codewhisperer/util/dependencyGraph/goDependencyGraph.test.ts

```diff
@@ -0,0 +1,180 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { GoDependencyGraph } from '../../../../codewhisperer/util/dependencyGraph/goDependencyGraph'
+import * as CodeWhispererConstants from '../../../../codewhisperer/models/constants'
+import assert from 'assert'
+import { join } from 'path'
+import sinon from 'sinon'
+import fs from 'fs'
+import { createMockDirentFile } from '../../testUtil'
+import { getTestWorkspaceFolder } from '../../../../testInteg/integrationTestsUtilities'
+
+describe('goDependencyGraph', () => {
+    const workspaceFolder = getTestWorkspaceFolder()
+    const appRoot = join(workspaceFolder, 'go1-zip')
+    const appCodePath = join(appRoot, 'main.go')
+    const goDependencyGraph = new GoDependencyGraph('go' satisfies CodeWhispererConstants.PlatformLanguageId)
+
+    describe('parseImport', () => {
```

Comment:
Avoid use of arrow functions and instead use `function () {...}` since `mocha` has some limitations: https://mochajs.org/#arrow-functions

Reply:
Didn't know that, thanks!